src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -964,6 +1086,7 @@ def get_image_features(
         output_hidden_states: Optional[bool] = None,
         return_dict: Optional[bool] = None,
         return_projected: Optional[bool] = True,
+        return_base_image_embeds: Optional[bool] = None,
```

Comment:
There's no need for this argument, adding it would make it less consistent with CLIP

Reply:
OWL-ViT is trained in two stages, the first stage consists of training a CLIP model and the second stage fine-tunes the whole model on the object detection task. I will need to add a similar argument to OwlViTForObjectDetection head if I remove this argument